creator-node/src/serviceRegistry.js

```diff
@@ -193,6 +195,72 @@ class ServiceRegistry {
     app.use('/health/bull', serverAdapter.getRouter())
   }
 
+  /**
+   * Wrap up current jobs in target queues under a max time. Used on death of app
+   */
+  async wrapUpQueueJobs() {
+    this.logInfo('Wrapping up jobs...')
+
+    const { queue: syncProcessingQueue } = this.syncQueue
+    const { queue: asyncProcessingQueue } = this.asyncProcessingQueue
+    const { queue: imageProcessingQueue } = this.imageProcessingQueue
+    const { queue: transcodingQueue } = this.transcodingQueue
+
+    const queues = [
```

Comment:
i think the only queue that is a concern for content unavail is manual sync queue jobs ([from this code](https://github.com/AudiusProject/audius-protocol/blob/vg-graceful-shutdown/creator-node/src/middlewares.js#L336))

i dont think other queues erroring are a problem